src/chrome-launcher.ts

```diff
@@ -405,11 +405,6 @@ class Launcher {
 
   destroyTmp() {
     return new Promise<void>(resolve => {
-      // Only clean up the tmp dir if we created it.
-      if (this.userDataDir === undefined || this.opts.userDataDir !== undefined) {
-        return resolve();
-      }
-
       if (this.outFile) {
         this.fs.closeSync(this.outFile);
         delete this.outFile;
```

Comment:
Makes sense we should always close these file handles when done. I'm curious though, what was the error you were seeing?